composer.json

```diff
@@ -19,14 +19,21 @@
         }
     ],
 
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stof/driver-testsuite"
+        }
+    ],
+
     "require": {
         "php":                       ">=5.3.1",
         "behat/mink":                "~1.7@dev",
         "instaclick/php-webdriver":  "~1.1"
     },
 
     "require-dev": {
-        "symfony/phpunit-bridge": "~2.7"
```

Comment:
I removed this here, because I handled it in ``mink/driver-testsuite`` (I want to force drivers to handle reporting of deprecations in their testsuite). Btw, the driver testsuite also allows using the Symfony 3 version, which is better.